service/matching/tasklist/task_list_manager.go

```diff
@@ -945,15 +945,7 @@ func (c *taskListManagerImpl) emitMisconfiguredPartitionMetrics() {
 	if c.config.NumReadPartitions() != c.config.NumWritePartitions() {
 		c.scope.UpdateGauge(metrics.TaskListReadWritePartitionMismatchGauge, 1)
 	}
-	pollerCount := len(c.pollerHistory.GetPollerInfo(time.Time{}))
-	if c.enableIsolation { // if isolation enabled, get the minimum poller count among the isolation groups
-		pollerCountsByIsolationGroup := c.pollerHistory.GetPollerIsolationGroups(time.Time{})
-		for _, count := range pollerCountsByIsolationGroup {
-			if count < pollerCount {
-				pollerCount = count
-			}
-		}
-	}
```

Comment:
These are isolation related logic. Are they still needed?

Reply:
With the tasklist partition <-> isolation group mapping that I'm currently implementing it's no longer accurate. We've disabled isolation broadly so it's not an issue in the mean time.